sdgx/models/components/optimize/sdv_ctgan/data_transformer.py

```diff
@@ -214,9 +238,12 @@ def inverse_transform(self, data, sigmas=None):
         Output uses the same type as input to the transform function.
         Either np array or pd dataframe.
         """
+
+        # TODO using pd.df.apply to increase performance.
```

Comment:
Remove if it's useless.

Reply:
That's a TODO for lastweek issue. Can I left it on code?